cmd/gateway/main.go

```diff
@@ -89,6 +90,11 @@ func init() {
 	configFlags.String("dapi-proxy-services", "", "specifies services exposed via _p endpoint proxies")
 	configFlags.Bool("debug", false, "enable debug mode")
 	configFlags.String("cpuprofile", "", "write cpu profile to a file")
+	configFlags.String("cb-pass-aws-id", "", "id of secret in aws sm storing couchbase server password")
```

Comment:
This should probably be something like cb-creds-aws-id, etc... since I believe they hold the credentials as a whole (username + password) rather than being strictly just the password.

Reply:
Done